src/pages/ListTableList/service.ts

```diff
@@ -0,0 +1,38 @@
+import request from '@/utils/request';
+import { TableListParams } from './data.d';
+
+export async function queryRule(params?: TableListParams) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'queryRule' to have a typedef](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4665582)